### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by andrevidela** [eczarny/spectacle#860]
The last commit (18b9d534decea200c8decf804d313c67953ebd6d) is from Jul 16, 2017, almost a year ago. There are 7 pending pull requests and over 90 open issues. Is this project still maintained? Should someone fork it? Should the owner change?

**Comment by a7hybnj2** [comment]
Oh my I haden't looked at those stats. I really like this utility and would hate to see it 'abandonware'.

**Comment by dimitrieh** [comment]
Would love to see  a new maintainer for this project, unless the old author picks it up again ;)

**Comment by Erowlin** [comment]
+1

**Comment by jamesoflol** [comment]
Same. I love this app, it's a critical part of my working environment. I would pay $$ for it!!

**Comment by am** [comment]
@jamesoflol you can pay: https://www.paypal.com/donate/?token=dYuGHWS_JLeuOWmHIBiat-E16Bt3oPo61hGXXJskTaeVGcq8opyEAIQoQZx9Gb85bZCi30&country.x=US&locale.x=US

😎

**Comment by Erowlin** [comment]
@am where did you get this URL? How to prove it's the correct email of @eczarny :)

**Comment by am** [comment]
Please... @Erowlin and @lespea I though the link was known by you, after-all is at the homepage of this project:

.1 Visit https://www.spectacleapp.com/
.2 Locate the phrase: `If you enjoy Spectacle please consider donating`
.3 Click donating link

As you can see this is the same page I refer. Truth is that the link I provided is hashed at runtime (when you click on the homepage a new hash is generated), so I guess that the best thing to do if you are willing to pay the money @eczarny deserves for his great app is to follow the steps above. 

Hope this clears any possible doubt regarding my last post. 

And finally:
![shut up and take my money](https://i.kym-cdn.com/photos/images/newsfeed/000/264/241/9e9.gif)

**Comment by Erowlin** [comment]
> Please... @Erowlin and @lespea I though the link was known by you, after-all is at the homepage of this project:
> 
> .1 Visit https://www.spectacleapp.com/
> .2 Locate the phrase: `If you enjoy Spectacle please consider donating`
> .3 Click donating link
> 
> As you can see this is the same page I refer. Truth is that the link I provided is hashed at runtime (when you click on the homepage a new hash is generated), so I guess that the best thing to do if you are willing to pay the money @eczarny deserves for his great app is to follow the steps above.
> 
> You could also check the source code to the homepage:
> https://github.com/eczarny/spectacle/blob/gh-pages/index.html#L53-L58
> 
> Hope this clears any possible doubt regarding my last post.
> 
> And finally:
> ![shut up and take my money](https://camo.githubusercontent.com/f4bb8847b39214c617cdacabc07521e08b67fb26/68747470733a2f2f692e6b796d2d63646e2e636f6d2f70686f746f732f696d616765732f6e657773666565642f3030302f3236342f3234312f3965392e676966)

Hey! 

Never said @eczarny didn't deserves any money, on the contrario, I was concerned about that link, thanks for clearing any doubt (I CMD + F on "Donate", I should've stopped at "Donat" :)).

**Comment by ryanpcmcquen** [comment]
So who is going to take it over and merge all the great PRs?

**Comment by ryanpcmcquen** [comment]
I switched to: https://github.com/fikovnik/ShiftIt

**Comment by hkroger** [comment]
I will try to contact @eczarny and see if I could help there. Merging PRs, testing and releasing shouldn't be too big of a problem.

**Comment by bigsee** [comment]
Would certainly be great to get this going again...

**Comment by ryanpcmcquen** [comment]
Someone could probably just recreate this with a HammerSpoon config:
https://github.com/Hammerspoon/hammerspoon

**Comment by mikep345678** [comment]
We've been grateful to be able to deploy Spectacle to every staff Mac in
our school district. Many of our teachers rely on it, primarily to move
windows between Macbooks and projectors.

Haven't tried ShiftIt but really don't want to have to retrain our staff to
new keystrokes!

Unfortunately, the Hammerspoon solution (
https://github.com/fikovnik/ShiftIt/wiki/The-Hammerspoon-Alternative) does
not appear to "scale" nearly as well as Spectacle... : (


Mike

**Comment by a7hybnj2** [comment]
@mikep345678 I did not like shiftit and they are going to be in the same boat soon: https://github.com/fikovnik/ShiftIt/issues/296

as of now spectacle still does everything I need it too. There are just a few wants that I have that for the time being I can live without.

I read about the hammerspoon method in the above link as well. I think I will give it a shot.

**Comment by a7hybnj2** [comment]
UPDATE: hammerspoon seems very capable and wouldn't be impossible to deploy in @mikep345678 case. The bindings could even be set similarly to spectacle.

The default hammerspoon bindings provided by https://github.com/fikovnik/ShiftIt/wiki/The-Hammerspoon-Alternative stink but that should be easy enough to fix and are customizable to user prefs.

**Comment by hkroger** [comment]
I tested miro windows manager as well and seems to work for me well.

**Comment by hkroger** [comment]
@ryanpcmcquen, that's right. It actually provides quite extensible way to achieve the same thing.

**Comment by rxhanson** [comment]
I've built a free open source window manager in Swift that's heavily based on Spectacle. It can be found at https://github.com/rxhanson/Rectangle. There's still a bit that's lacking from Spectacle, like screen thirds and localization, but the core of it is all there. Feel free to give it a try!
